portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -54,7 +61,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         response.sendRedirect("/comments.html");
     }
     private Comment getComment(HttpServletRequest request) {
-        Comment comment = new Comment(request.getParameter(NAME_PARAMETER), request.getParameter(COMMENT_PARAMETER));
+        long timestamp = System.currentTimeMillis();
+        Comment comment = new Comment(request.getParameter(NAME_PARAMETER), request.getParameter(COMMENT_PARAMETER), timestamp);
+        newEntity(comment);
         return comment;
     }
+    private void newEntity(Comment comment) {
```

Comment:
This function is just name `newEntity` which would lead me to believe it returns `Entity`; it has the side effect of storing in the `Datastore` that isn't obvious. Maybe split into some single purpose functions. 

1. Function for creating a comment.
2. Function for creating an entity from a comment.
3. Store that entity.